README.md

```diff
@@ -1,16 +1,176 @@
-# Title
+# Print, Puts, and Return Values
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Define and distinguish `puts` from `print`
+- Identify implicit return values in Ruby syntax
+- Recognize the explicit `return` keyword
 
 ## Introduction
 
-## SWBAT 1
+There are a few ways to output text in Ruby. In previous lessons, we've used the
+keyword `puts` in order to display text in the console, but we can also use
+`prints`. How are these different? And when should you use one or the other?
 
-## SWBAT 2
+Displaying text in the console is not the only type of _return value_ available
+to us. When writing code, sometimes we'll want our methods to evaluate code and
+_return_ a value, but not display it.
+
+We'll cover how the `puts` and `print`commands display Ruby code to the console,
+and the concept of return values in Ruby.
+
+## Define and distinguish `puts` from `print`
+
+The `puts` (short for "out**put s**tring") and `print` commands are both used to
+display in the console the results of evaluating Ruby code. The primary
+difference between them is that `puts` adds a new line after executing, and
+`print` does not.
+
+```ruby
+3.times { print "Hello!" }
+# > Hello!Hello!Hello!
+
+3.times { puts "Hello!" }
+# > Hello!
+# > Hello!
+# > Hello!
+```
+
+The methods `puts` and `print` are a great way to explicitly tell the program to
```

Comment:
Simpler: The methods `puts` and `print` tell the program to display specific information